### PR TITLE
feat: list connected dapps

### LINKED ISSUE #120: ️🛠 - List of connected DApps
### Description

For the time being until we have some professional designs, add a button on the top navigation “Apps” which will open a modal with all connected DApps. Include the current “Connect” button into the top of the modal as “Connect new DApp” 

### Requirements

- [x]  Add “Apps” button in top navigation
- [x]  Modal with list of connected pairings
- [x]  Pairing should show the icon + name
- [x]  Move “Connect” button into modal as “Connect new DApp”

### Checklist

- [X] I have linked the correct milestone and/or project(s)
- [X] I have added the appropriate labels
- [ ] I have assigned this task to the correct people

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/drawers/dapp-config/components/DappCard.svelte [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    export let connectedDapp: IConnectedDapp = undefined
+
+    function onCardClick(): void {
+        // TODO
+    }
+    const localeKey = 'views.dashboard.drawers.dApps.dAppsList.state.'
+    $: pill = connectedDapp.active
+        ? {
+              data: '• ' + localize(localeKey + 'connected'),
```
Comment (nicole-obrien) [inline packages/desktop/views/dashboard/drawers/dapp-config/components/DappCard.svelte]: these localise are very generic and we already have them under general?

**Review by Tuditi — CHANGES_REQUESTED** [review]
Looks good. The connection status with a Dapp is out of sync. 

E.g.: If I click on disconnect from the app, the status for the dapp card isn't updated. 

I think it would be good to add it in this PR, but let me know if you created a separate task for it, then I can approve this one.

File: packages/shared/src/lib/auxiliary/wallet-connect/stores/connected-dapps.store.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Writable, get, writable } from 'svelte/store'
+import { getWalletClient } from './wallet-client.store'
+import { IConnectedDapp } from '../interface/connected-dapp.interface'
+
+export const connectedDapps: Writable<IConnectedDapp[]> = writable([])
```
Comment (nicole-obrien) [inline packages/shared/src/lib/auxiliary/wallet-connect/stores/connected-dapps.store.ts:5]: Don't we need a persisted dapps store for each profile? And then connected dapps is only when that profile is connected. Currently I am getting all dapps across all profiles

File: packages/shared/src/lib/auxiliary/wallet-connect/handlers/onSessionProposal.handler.ts [hunk 3]
```diff
@@ -23,5 +23,7 @@ export function onSessionProposal(sessionProposal: Web3WalletTypes.SessionPropos
         },
     })
 
-    void getWalletClient()?.approveSession({ id, namespaces: approvedNamespaces })
+    void getWalletClient()
```
Comment (nicole-obrien) [inline packages/shared/src/lib/auxiliary/wallet-connect/handlers/onSessionProposal.handler.ts]: I don't mind the .then but its going against convention

**Review by nicole-obrien — DISMISSED** [review]

File: packages/shared/src/lib/auxiliary/wallet-connect/stores/connected-dapps.store.ts (hunk 2 above)
Comment (MarkNerdi) [reply packages/shared/src/lib/auxiliary/wallet-connect/stores/connected-dapps.store.ts:5]: https://github.com/bloomwalletio/bloom/issues/466

**Review by Tuditi — APPROVED** [review]

**Pull request merged**